app/bundles/AssetBundle/Entity/Download.php

```diff
@@ -66,6 +66,32 @@ class Download
      */
     private $email;
 
+    /**
+     * @var string
+     */
+    private $utmCampaign;
+
+    /**
+     * @var string
+     */
+    private $utmContent;
+
+    /**
+     * @var string
+     */
+    private $utmMedium;
+
+    /**
+     * @var string
+     */
+    private $utmSource;
+
+    /**
+     * @var string
+     */
+    private $utmTerm;
```

Comment:
Let's use typed properties in new code at least. All entity properties can be null if they are not set directly through the constructor.
```suggestion
    private ?string $utmCampaign;

    private ?string $utmContent;

    private ?string $utmMedium;

    private ?string $utmSource;

    private ?string $utmTerm;
```